cmd/spiffe-helper/main.go

```diff
@@ -13,39 +13,43 @@ import (
 )
 
 func main() {
-	// 0. Load configuration
-	// 1. Create Sidecar
-	// 2. Run Sidecar's Daemon
-
 	configFile := flag.String("config", "helper.conf", "<configFile> Configuration file path")
-	exitWhenReady := flag.Bool("exitWhenReady", false, "Exit once the requested objects are retrieved")
+	disableDaemonMode := flag.Bool("disable_daemon_mode", false, "Exit once the requested objects are retrieved")
```

Comment:
Isn't the config `daemon_mode`, not `disable_daemon_mode`?

Reply:
Changed to daemon mode